core/test/financial-templates/Liquidatable.js

```diff
@@ -1231,6 +1231,89 @@ contract("Liquidatable", function(accounts) {
   });
 
   describe("Weird Edge cases", () => {
+    it("Liquidating 0 tokens always results in a successful Dispute", async () => {
+      // TODO: Liquidations for 0 tokens should be blocked but they are not. They will harmlessly always cause a successful dispute because
```

Comment:
I guess I left this TODO because we should really not even enable this situation in the first place. But we can remove this test when we block this behavior.

Reply:
cool. I saw you also updated issue #1553 so it's recorded there 👍